unstructured_inference/inference/layout.py

```diff
@@ -427,6 +455,8 @@ def from_image(
         """Creates a PageLayout from an already-loaded PIL Image."""
         analysis = kwargs.get("analysis", False)
         supplement_with_ocr_elements = kwargs.get("supplement_with_ocr_elements", True)
+        extract_images_in_pdf = kwargs.get("extract_images_in_pdf", False)
+        image_output_dir_path = kwargs.get("image_output_dir_path", None)
```

Comment:
You didn't start this pattern, but why don't we just move all these to the signature.

Reply:
By "all these" I mean `analysis`, `supplement_with_ocr_elements`, `extract_images_in_pdf`, and `image_output_dir_path`.

Reply:
Sounds good!